torch/_dynamo/variables/builder.py

```diff
@@ -1393,6 +1394,8 @@ def build_key_value(i, k, v):
                 self_obj,
                 source=self.source,
             )
+        elif istype(value, object):
```

Comment:
nit: Since object is a very generic type, could we move it later in this if/elif logic? (similar to SourcelessBuilder)